docs/userguide.md

```diff
@@ -305,6 +305,27 @@ To cleanup the example simply delete the namespace:
 kubectl delete ns test-namespace
 ```
 
+### Troubleshooting
+
+If you found the federated resources were not propagated to the member clusters, you can always
+use the following command to get some failure reason by checkig the `Events`.
```

Comment:
s/to get some failure reason by checkig the Events/to view Events which may aid in diagnosing the problem/